packages/backend/src/setup/createRpcTransactionUpdaters.ts

```diff
@@ -83,3 +84,24 @@ export function createEthereumTransactionUpdater(
   )
   return updater
 }
+
+export function createStarkNetTransactionUpdater(
+  config: TransactionCountSyncConfig,
+  blockTransactionCountRepository: BlockTransactionCountRepository,
+  clock: Clock,
+  logger: Logger,
+) {
+  const client = new StarkNetClient({ callsPerMinute: 60 * 5 })
```

Comment:
Shouldn't callsPerMinute come from config?

Reply:
Have you tested how many calls per minute can the provider handle?

Reply:
Yes, this value is causing `Too Many Requests` error sometimes, but it keeps on going.